db/migrate/20161124211634_create_votes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateVotes < ActiveRecord::Migration
+  def change
+    create_table :votes do |t|
+      t.references :user, null: false
+      t.references :voteable, polymorphic: true, index: true
+      
```

Comment:
I was looking at the documentation for polymorphic associations and it looked like if it is set up this way, activerecord handles the other end. There are validations in the models for the polymorphic relationships.